code/intermediate/tutorial11-normals/src/model.rs

```diff
@@ -220,6 +220,9 @@ impl Model {
                 let r = 1.0 / (delta_uv1.x * delta_uv2.y - delta_uv1.y * delta_uv2.x);
                 let tangent = (delta_pos1 * delta_uv2.y - delta_pos2 * delta_uv1.y) * r;
                 let bitangent = (delta_pos2 * delta_uv1.x - delta_pos1 * delta_uv2.x) * r;
+                // We flip the bitangent to enable right-handed normal
```

Comment:
@sotrh Are you happy with this comment and the use of the `flipped_bitangent` var?

Reply:
updated to not introduce new var.